tests/pages/NouvelleAction.page.test.tsx

```diff
@@ -199,8 +196,72 @@ describe('NouvelleAction', () => {
           expect(actionsService.createAction).not.toHaveBeenCalled()
         })
 
+        it("affiche un message d'erreur quand type d’action prédéfinie est vide", async () => {
+          // When
+          expect(selectAction).toBeInTheDocument()
+          await userEvent.click(selectAction)
+          await userEvent.tab()
+
+          // Then
+          expect(selectAction.value).toEqual('')
+          expect(
+            screen.getByText('Le champ “Titre” est vide. Renseignez un titre.')
```

Comment:
à modifier cf ticket(qui comporte une coquille aussi, car c'est une action prédéfinie, pas un rendez-vous).
```suggestion
            screen.getByText('"Le champ “Action prédéfinie” est vide. Sélectionnez un type...')
```